python/servo/mutation/test.py

```diff
@@ -42,8 +42,11 @@ def mutation_test(file_name, tests):
             if mutated_line != -1:
                 logging.info("Mutated {0} at line {1}".format(file_name, mutated_line))
                 logging.info("compiling mutant {0}:{1}".format(file_name, mutated_line))
-                if subprocess.call('python mach build --release', shell=True, stdout=DEVNULL):
+                test_command = "python mach build --release"
+                if subprocess.call(test_command, shell=True, stdout=DEVNULL):
                     logging.error("Compilation Failed: Unexpected error")
+                    logging.error("Failed: while running `{0}`".format(test_command))
```

Comment:
IMO, we can know we got compilation failed from the messages like following.
Maybe we don't need to log this?
cc @jdm 

```
2017-12-14 19:01:46,142 INFO: compiling mutant components/script/dom/range.rs:803
   Compiling script v0.0.1 (file:///home/servo/buildbot/slave/linux-rel-nogate/build/components/script)
error[E0425]: cannot find value `text` in this scope
   --> components/script/dom/range.rs:804:13
    |
804 |             text.ReplaceData(start_offset,
    |             ^^^^ did you mean `next`?
help: possible candidates are found in other modules, you can import them into scope
    |
5   | use dom::bindings::codegen::Bindings::RequestBinding::RequestBinding::text;
    |
5   | use dom::bindings::codegen::Bindings::ResponseBinding::ResponseBinding::text;
    |

error: aborting due to previous error

error: Could not compile `script`.

To learn more, run the command again with --verbose.
```

Reply:
The user would only know the build is failed.
But they would know what command is only if they go to see the source code.
Whatever, leave it or not, i though both are OK.